spacy/cli/debug_data.py

```diff
@@ -986,10 +986,18 @@ def _get_kl_divergence(p: Counter, q: Counter) -> float:
     return total
 
 
+def _format_number(number: Union[int, float], ndigits: int = 2) -> str:
```

Comment:
Maybe this could go into `spacy.cli.util`?

Reply:
Yeah, my reservation is that there's no other cli commands that use this (yet). I only put it here because we have other specific `debug` and `span` formatting code in this section, so now it's right next to that. I think if we had another PR where this function was used more generally than for `debug data (spans)`, it would make more sense to pull it out into `util`.

Reply:
I think it would be good to pull this out though. It's a little "chicken or egg": If you don't put this in `util` now, we might not realise this function is there when working on other CLI commands, thus we'd end up not reusing it.

Reply:
Done!